tests/devices/test_default_mixed_autograd.py

```diff
@@ -0,0 +1,585 @@
+        assert circuit1.gradient_fn == "backprop"
+        assert circuit2.gradient_fn is qml.gradients.param_shift
+
+        res = circuit1(p)
+
+        assert np.allclose(res, circuit2(p), atol=tol, rtol=0)
+
+        grad_fn = qml.jacobian(circuit1, 0)
+        res = grad_fn(p)
+        assert np.allclose(res, qml.jacobian(circuit2)(p), atol=tol, rtol=0)
+
+    def test_state_differentiability(self, tol):
+        """Test that the device state can be differentiated"""
+        dev = qml.device("default.mixed", wires=1)
+
+        @qml.qnode(dev, diff_method="backprop", interface="autograd")
+        def circuit(a):
+            qml.RY(a, wires=0)
+            return qml.expval(qml.PauliZ(0))
+
+        a = np.array(0.54, requires_grad=True)
+
+        def cost(a):
+            """A function of the device quantum state, as a function
+            of input QNode parameters."""
+            circuit(a)
+            res = np.abs(dev.state) ** 2
+            return res[1][1] - res[0][0]
+
+        grad = qml.grad(cost)(a)
+        expected = np.sin(a)
+        assert np.allclose(grad, expected, atol=tol, rtol=0)
+
+    def test_prob_differentiability(self, tol):
+        """Test that the device probability can be differentiated"""
+        dev = qml.device("default.mixed", wires=2)
+
+        @qml.qnode(dev, diff_method="backprop", interface="autograd")
+        def circuit(a, b):
+            qml.RX(a, wires=0)
+            qml.RY(b, wires=1)
+            qml.CNOT(wires=[0, 1])
+            return qml.probs(wires=[1])
+
+        a = np.array(0.54, requires_grad=True)
+        b = np.array(0.12, requires_grad=True)
+
+        def cost(a, b):
+            prob_wire_1 = circuit(a, b)
+            return prob_wire_1[1] - prob_wire_1[0]
+
+        res = cost(a, b)
+        expected = -np.cos(a) * np.cos(b)
+        assert np.allclose(res, expected, atol=tol, rtol=0)
+
+        grad = qml.grad(cost)(a, b)
+        expected = [np.sin(a) * np.cos(b), np.cos(a) * np.sin(b)]
+        assert np.allclose(grad, expected, atol=tol, rtol=0)
+
+    def test_backprop_gradient(self, tol):
+        """Tests that the gradient of the qnode is correct"""
+        dev = qml.device("default.mixed", wires=2)
+
+        @qml.qnode(dev, diff_method="backprop", interface="autograd")
+        def circuit(a, b):
+            qml.RX(a, wires=0)
+            qml.CRX(b, wires=[0, 1])
+            return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
+
+        a = np.array(-0.234, requires_grad=True)
+        b = np.array(0.654, requires_grad=True)
+
+        res = circuit(a, b)
+        expected_cost = 0.5 * (np.cos(a) * np.cos(b) + np.cos(a) - np.cos(b) + 1)
+        assert np.allclose(res, expected_cost, atol=tol, rtol=0)
+
+        res = qml.grad(circuit)(a, b)
+        expected_grad = np.array(
+            [-0.5 * np.sin(a) * (np.cos(b) + 1), 0.5 * np.sin(b) * (1 - np.cos(a))]
+        )
+        assert np.allclose(res, expected_grad, atol=tol, rtol=0)
+
+    @pytest.mark.skip
```

Comment:
How come this is being skipped?

Reply:
Oh, I forgot to add it back in after our previous discussion on handling this.